### ISSUE TITLE
Popover trigger is not working

### ISSUE THREAD

**Issue by jiayihu** [reactstrap/reactstrap#1333]
- components: `Popover`
- reactstrap version `#6.5.0`
- import method `umd/csj/es`
- react version `16.5.2`
- bootstrap version `#x.x.x`

### What is happening?

Popover is not toggled when clicking/focusing the button. If I inspect the component and set `isOpen` prop with React devtools than the popover appears, but the button doesn't seem to have effects on the popover. `id` and `target` are fine, if they don't match there is an error.

### What should be happening?

It should toggle.

### Steps to reproduce issue

Documentation example doesn't work in Codesandbox.

[https://codesandbox.io/s/8zyw4ny7k8](https://codesandbox.io/s/8zyw4ny7k8)

### Code

```jsx
import React from 'react';
import { Button, Popover, PopoverHeader, PopoverBody } from 'reactstrap';

export default class Example extends React.Component {
  constructor(props) {
    super(props);

    this.toggle = this.toggle.bind(this);
    this.state = {
      popoverOpen: false
    };
  }

  toggle() {
    this.setState({
      popoverOpen: !this.state.popoverOpen
    });
  }

  render() {
    return (
      <div>
        <Button id="Popover1" type="button">
          Launch Popover
        </Button>
        <Popover placement="bottom" isOpen={this.state.popoverOpen} target="Popover1" toggle={this.toggle}>
          <PopoverHeader>Popover Title</PopoverHeader>
          <PopoverBody>Sed posuere consectetur est at lobortis. Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum.</PopoverBody>
        </Popover>
      </div>
    );
  }
}
```

**Comment by ssarkar-auro** [comment]
facing same issue.

**Comment by LukasGasior1** [comment]
Same here. Have you, by any chance, found a solution @jiayihu?

**Comment by jiayihu** [comment]
Yup, just add `onClick={this.toggle}` handler to the Button, but I'm leaving the issue open because I want to know if it is the actual solution or just a workaround. Documentation doesn't mention about it and `target` prop would be useless.

**Comment by kinke** [comment]
New v7 has a trigger prop and defaults to `click`, so this should do what you want.

**Comment by Amir-Pasha-Bagheri** [comment]
I have version 9 and i have same issue , someone help !

**Comment by anasbn3issa** [comment]
I have reactstrap v9 
and this worked for me , 

toggle function to set Open on click 
and used isOpen prop i found on documentation 
(@jiayihu  ur code on the question actually helped me ) 
`export default function OwnerPopover({ owner }) {
    const [open, setOpen] = React.useState(false);
    const toggle = () => setOpen(!open);
  return (
    <div>
      <Button id="Popover1" type="button" onClick={toggle} >
        Launch Popover
      </Button>
      <Popover flip target="Popover1" isOpen={open}>
        <PopoverHeader>Popover Title</PopoverHeader>
        <PopoverBody>
          Sed posuere consectetur est at lobortis. Aenean eu leo quam.
          Pellentesque ornare sem lacinia quam venenatis vestibulum.
        </PopoverBody>
      </Popover>
    </div>
  );
}`
